src/test_driver/esp32/sdkconfig.defaults

```diff
@@ -31,3 +31,6 @@ CONFIG_ESP_MAIN_TASK_STACK_SIZE=32768
 
 #enable BT
 CONFIG_BT_ENABLED=y
+
+#Disable Hardware SHA - Workaround to issue 8702
```

Comment:
Can someone confirm if this change is required in `src/test_driver/esp32`, @andy31415 ?

Reply:
@tcarmelveilleux has investigated some of these. My understanding is that until case supports single sha at a time, hw sha cannot be used (as it supports only one stream at a time)